requirements/edx/base.txt

```diff
@@ -63,7 +63,7 @@ coreapi==2.3.3            # via django-rest-swagger, openapi-codec
 coreschema==0.0.4         # via coreapi
 cryptography==2.3.1
 cssutils==1.0.2           # via pynliner
-ddt==0.8.0
+ddt==1.2.0
```

Comment:
If you removed ddt from base.in, shouldn't it be gone from base.txt?  Was this regenerated?

Reply:
**base.text** has been regenerated. The **ddt** is still required by **xblock-drag-and-drop-v2** and **xblock-poll** :roll_eyes:

Reply:
ok!